### ISSUE TITLE
Live pagination issues while using Tab key navigation

### ISSUE THREAD

**Issue by yurigenin** [inovua/reactdatagrid#130]
- what edition are you using - enterprise
- version for `@inovua/reactdatagrid-enterprise` - 4.1.15

Relevant code or config

```javascript
import React, { useState, useCallback } from 'react'
import ReactDataGrid from '@inovua/reactdatagrid-enterprise'
import '@inovua/reactdatagrid-enterprise/index.css'

import CheckBox from '@inovua/reactdatagrid-community/packages/CheckBox'
import Button from '@inovua/reactdatagrid-community/packages/Button'

import DATASET_URL from './DATASET_URL'

const gridStyle = { minHeight: 400, marginTop: 10 }

const columns = [
  { name: 'id', header: 'Id', defaultVisible: false, type: 'number', maxWidth: 40 },
  { name: 'firstName', defaultWidth: 400, header: 'First Name' },
  { name: 'lastName', defaultWidth: 300, header: 'Last Name' },
  { name: 'email', groupBy: false, defaultWidth: 300, header: 'Email', editable: false }
]

const loadData = ({ skip, limit }) => {
  return fetch(DATASET_URL + '?skip='+skip + '&limit='+limit).then(response => {
    const totalCount = response.headers.get('X-Total-Count');   
    return response.json().then(data => {
      return { data, count: parseInt(totalCount) };
    })
  })
}

const App = () => {
  const [livePagination, setLivePagination] = useState(true);

  const dataSource = useCallback(loadData, [])

  return (
    <div>
      <div style={{ marginBottom: 20 }}>
        <CheckBox checked={livePagination} onChange={setLivePagination} >
          Live pagination
        </CheckBox>
      </div>
      <ReactDataGrid
        idProperty="id"
        key={'grid-' + livePagination}
        style={gridStyle}
        columns={columns}
        livePagination={livePagination}
        pagination
        sortable={false}
        dataSource={dataSource}
        limit={12}
        editable
      />
    </div>
  );
}

export default () => <App />
```

In this example as you double click on the first cell of the first row and start hitting the Tab key eventually the grid only loads two pages and afterwards the editing exits so that you can never get to edit the Marcelo row.

**Comment by inovua-admin** [comment]
Fixed in `v4.1.16`

**Comment by yurigenin** [comment]
Other issue
